src/service/poc_mobile.proto

```diff
@@ -664,3 +666,27 @@ verified_service_provider_boosted_rewards_banned_radio_ingest_report_v1 {
   // epoch
   uint64 timestamp = 3;
 }
+
+message subscriber_mapping_event_req_v1 {
+  /// The id of the discovery mapping enabled subscriber
+  bytes subscriber_id = 1;
+  /// The accumulated mapping points the subscriber has earned this epoch
+  uint64 total_reward_points = 2;
+  /// Unix timestamp in seconds of when the mapping event was generated
+  uint64 timestamp = 3;
+  /// Pubkey of verification mapping event service
+  bytes pub_key = 6;
+  /// Signed payload of the verification mapping event service
+  bytes signature = 7;
+}
+
+message subscriber_mapping_event_res_v1 { string id = 1; }
+
+message verified_subscriber_mapping_event_v1 {
```

Comment:
i dont understand what this msg is for ? You should have another msg that is outputted from the ingestor which wraps the original subscriber_mapping_event_req_v1 msg...see my original proto branch here for reference: https://github.com/helium/proto/blob/andymck/subscriber-mapping-support/src/service/poc_mobile.proto#L228

Reply:
ok I made it more like your branch